packages/esm-patient-chart-app/src/clinical-views/components/encounter-tile/tile.scss

```diff
@@ -0,0 +1,96 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.title {
+  @include type.type-style('productive-heading-03');
+}
+
+.title:after {
+  content: '';
+  display: block;
+  width: layout.$spacing-07;
+  padding-top: layout.$spacing-01;
+  border-bottom: layout.$spacing-03 solid $brand-teal-01;
+}
+
+.tileTitle {
+  @include type.type-style('label-01');
+  display: block;
+  padding-top: layout.$spacing-05;
+}
+
+.columnContainer {
+  margin: 0px 0px layout.$spacing-06 layout.$spacing-06;
+  display: flex;
+  flex-direction: row;
+}
+
+.tileBoxColumn {
+  width: 100%;
+  display: 'flex';
+  flex-direction: 'row';
+}
+
+.tileSubTitle {
+  @include type.type-style('body-compact-01');
+}
+
+.tileValue {
+  @include type.type-style('body-long-01');
+  margin: layout.$spacing-05 layout.$spacing-10 layout.$spacing-06 0;
+}
+
+.tileBox {
+  width: 25%;
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+}
+
+.desktopHeading {
+  h4 {
+    @include type.type-style('heading-compact-02');
+    color: $text-02;
+  }
+}
+
+.tabletHeading {
+  h4 {
+    @include type.type-style('heading-03');
+    color: $text-02;
+  }
+}
+
+.desktopHeading,
+.tabletHeading {
+  text-align: left;
+  text-transform: capitalize;
+  margin-bottom: layout.$spacing-05;
+
+  h4:after {
+    content: '';
+    display: block;
+    width: layout.$spacing-07;
+    padding-top: 0.188rem;
+    border-bottom: 0.375rem solid var(--brand-03);
+  }
+}
+
+.heading:after {
+  content: '';
+  display: block;
+  width: layout.$spacing-07;
+  padding-top: 0.188rem;
```

Comment:
Seems like this could just be `layout.$spacing-01`, not exactly the same, but close enough, unless we have a really strong reason for using non-standard values?